RELEASE_NOTES.md

```diff
@@ -1,2 +1,4 @@
 
 # Next Release
+
+- [#58](https://github.com/iiasa/ixmp/pull/58): Add MacOSX support in CI
```

Comment:
@gidden do you think release notes should be committed to a file in repo instead of having draft on releases tab of repository? e.g. https://github.com/iiasa/scenario-explorer-new/releases